src/core/manager.c

```diff
@@ -3871,6 +3871,9 @@ static void manager_notify_finished(Manager *m) {
 
         bus_manager_send_finished(m, firmware_usec, loader_usec, kernel_usec, initrd_usec, userspace_usec, total_usec);
 
+        if (MANAGER_IS_SYSTEM(m) && detect_container() <= 0)
+                watchdog_report_if_missing();
```

Comment:
this is racy of course. the initial transaction might be over very quickly, and the watchdog drivers might take quite long to initialize, if they are kmods and connected via slow wiring..

That might not be a reason not to do this, but it does make me concerned about making this anything beyond LOG_NOTICE

Reply:
I think that if the system is *configured* to use a watchdog, then it's iffy it it doesn't get detected. Let's make it a warning for now. If poeple report that they have systems where this doesn't hold, we can always downgrade.